nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,594 @@
+        argstr="--no-reduce-fov", desc="do not reduce FoV to encompass mask"
+    )
+
+    reduce_fox_eqodd = traits.Bool(
+        argstr="--reduce-fox-eqodd",
+        desc="reduce FoV to encompass mask but force nc=nr and ns to be odd",
+    )
+
+    contrast = InputMultiPath(
+        File(exists=True), argstr="--C %s...", desc="contrast file"
+    )
+
+    default_seg_merge = traits.Bool(
+        argstr="--default-seg-merge", desc="default schema for merging ROIs"
+    )
+
+    merge_hypos = traits.Bool(
+        argstr="--merge-hypos", desc="merge left and right hypointensites into to ROI"
+    )
+
+    merge_cblum_wm_gyri = traits.Bool(
+        argstr="--merge-cblum-wm-gyri",
+        desc="cerebellum WM gyri back into cerebellum WM",
+    )
+
+    tt_reduce = traits.Bool(
+        argstr="--tt-reduce", desc="reduce segmentation to that of a tissue type"
+    )
+
+    replace = traits.Tuple(
+        traits.Int,
+        traits.Int,
+        argstr="--replace %i %i",
+        desc="Id1 Id2 : replace seg Id1 with seg Id2",
+    )
+
+    rescale = traits.List(
+        argstr="--rescale %s...",
+        desc="Id1 <Id2...>  : specify reference region(s) used to rescale (default is pons)",
+    )
+
+    no_rescale = traits.Bool(
+        argstr="--no-rescale",
+        desc="do not global rescale such that mean of reference region is scaleref",
+    )
+
+    scale_refval = traits.Float(
+        argstr="--scale-refval %f",
+        desc="refval : scale such that mean in reference region is refval",
+    )
+
+    _ctab_inputs = ("color_table_file", "default_color_table", "gca_color_table")
+    color_table_file = File(
+        exists=True,
+        argstr="--ctab %s",
+        xor=_ctab_inputs,
+        desc="color table file with seg id names",
+    )
+
+    default_color_table = traits.Bool(
+        argstr="--ctab-default",
+        xor=_ctab_inputs,
+        desc="use $FREESURFER_HOME/FreeSurferColorLUT.txt",
+    )
+
+    tt_update = traits.Bool(
+        argstr="--tt-update",
+        desc="changes tissue type of VentralDC, BrainStem, and Pons to be SubcortGM",
+    )
+
+    lat = traits.Bool(argstr="--lat", desc="lateralize tissue types")
+
+    no_tfe = traits.Bool(
+        argstr="--no-tfe",
+        desc="do not correct for tissue fraction effect (with --psf 0 turns off PVC entirely)",
+    )
+
+    no_pvc = traits.Bool(
+        argstr="--no-pvc",
+        desc="turns off PVC entirely (both PSF and TFE)",
+    )
+
+    segpvfres = traits.Float(
+        argstr="--segpvfres %f",
+        desc="set the tissue fraction resolution parameter (def is 0.5)",
+    )
+
+    rbv = traits.Bool(argstr="--rbv", requires=["subjects_dir"], desc="perform RBV PVC")
+
+    rbv_res = traits.Float(
+        argstr="--rbv-res %f",
+        desc="voxsize : set RBV voxel resolution (good for when standard res takes too much memory)",
+    )
+
+    mg = traits.List(argstr="--id %s...", desc="Manually specify segmentation ids")
```

Comment:
Name doesn't seem to match arg, and I can't find `--id` in docs. I do see:

`"   --mg gmthresh RefId1 RefId2 ...: perform Mueller-Gaertner PVC, gmthresh is min gm pvf bet 0 and 1\n"`

I suspect you want something like:

```suggestion
    mg = traits.Tuple(
        traits.Float,
        traits.List(traits.String),
        argstr="--mg %g %s...",
        desc="gmthresh RefId1 RefId2 ...: perform Mueller-Gaertner PVC, gmthresh is min gm pvf bet 0 and 1")
```

That said, you might need to patch `_format_arg()` to correctly handle this. I'm not sure if `...` will do what you want on its own.